backend/experiment/actions/final.py

```diff
@@ -1,45 +1,174 @@
+                               {"image": "/static/logo.png", "link": "https://example.com"}.
+    """
+
+    view: str
+    score: float
+    rank: str | None
+    final_text: str | None
+    button: ButtonDto | None
+    points: str
+    action_texts: dict
+    title: str
+    social: SocialMediaConfigDto | None
+    show_profile_link: bool
+    show_participant_link: bool
+    feedback_info: dict | None
+    participant_id_only: bool
+    logo: LogoDto | None
+
+
 class Final(BaseAction):  # pylint: disable=too-few-public-methods
     """
-    Provide data for a final view
+    Provide data for a "final" view, typically shown at the end of an experiment or session.
+
+    This view displays the participant's final score and, optionally, their rank or performance category.
+    It can also present navigation elements, such as a "Play again" button or links to other parts of the site.
+    Branding elements like a logo or social sharing options can be included to enhance user engagement.
+    A feedback section may also be provided if `feedback_info` is supplied.
+
+    The returned data aligns with `FinalActionDto`, ensuring type consistency and making the structure
+    clear for both developers and documentation readers. It can be consumed by a frontend component
+    (e.g.,
+    [Final.tsx](https://amsterdam-music-lab.github.io/MUSCLE/storybook/?path=/story/final--default)) to render the final screen.
+
+    Args:
+        session (Session): The current session object associated with the participant.
+        title (str): The title displayed at the top of the final view. Defaults to a localized "Final score".
+        final_text (str): An optional concluding message (e.g., "Thanks for participating!").
+        button (ButtonDto): Optional call-to-action button configuration. For example:
+                            {"text": "Play again", "link": "/{experiment_slug}"}.
+        points (str): The label for the score units (e.g., "points"). Defaults to a localized "points".
+        rank (str): The participant's rank (e.g., "GOLD"). If not provided, no rank is displayed.
+        show_profile_link (bool): If True, display a link to the participant's profile.
+        show_participant_link (bool): If True, display a participant-related link or information.
+        show_participant_id_only (bool): If True, only the participant ID is shown, without a link.
+        feedback_info (dict): Optional dictionary containing feedback-related data. For example:
+                              {"header": "Feedback", "prompt": "Tell us what you think", "button_text": "Submit"}.
+        total_score (float): Explicit final score. If None, this is derived from the session.
+        logo (LogoDto): Optional logo configuration for branding. For example:
+                        {"image": "/static/logo.png", "link": "https://example.com"}.
 
-    Relates to client component: Final.js
+    Attributes:
```

Comment:
Why specify `Args` *and* `Attributes`?

Reply:
I went full monty as I didn't know what the documentation conventions are for our project and the fact that Args & Attributes are not the same. But I will remove Attributes now